modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -388,51 +388,51 @@ class StandardHeader(
     }
 
     sendKeywords(id, inst, hs, List(
+      buildString(obsObject.orDefault, "OBJECT"),
       buildString(obsReader.getObsType, "OBSTYPE"),
       buildString(obsReader.getObsClass, "OBSCLASS"),
       buildString(obsReader.getGemPrgId, "GEMPRGID"),
       buildString(obsReader.getObsId, "obsid"),
       buildString(obsReader.getDataLabel, "DATALAB"),
+      buildString(stateReader.getObserverName, "OBSERVER"),
```

Comment:
Nice, would this solve the issue that the names are not being applied to the FITS keywords?